### PR TITLE
Add nvmrc to specify node version

### PR DESCRIPTION
**Pull request by junli1026**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

I often met yarn error because my node version is too old,  so added the nvm to specify node version.

## Changelog

- Improvement

## Related Issues

Fixes #issue

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: website/databend/README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ This website is built using [Docusaurus 2](https://docusaurus.io/), a modern sta
 Local preview
 
 ```shell
+nvm use
```
[the remark this answers is not in the document]
Comment (junli1026) [reply website/databend/README.md:10]: sure will address

File: website/databend/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v16.13.0
```
[the remark this answers is not in the document]
Comment (junli1026) [reply website/databend/.nvmrc:1]: I will address, but I think both should work.

**Review by Xuanwo — APPROVED** [review]

**Comment by Xuanwo** [comment]
The failed is not related with this PR, please ignore.

**Comment by BohuTANG** [comment]
Test failure not related to this PR, merged.
Thank you @junli1026

**Pull request merged**